src/run_server.sh

```diff
@@ -1,27 +1,40 @@
 #!/usr/bin/env bash
 #######################################################################
 #   Author: Renegade-Master
+#	Contributor: JohnEarle
 #   Description: Install, update, and start a Dedicated Project Zomboid
 #       instance.
 #######################################################################
 
 # Set to `-x` for Debug logging
 set +x
+# Start the Server for the first time to generate configuration files for editing
+function start_server_initial() {
+	clear
+    printf "\n### Starting Project Zomboid Server...\n"
+    timeout 30 "$BASE_GAME_DIR"/start-server.sh \
+        -adminusername "$ADMIN_USERNAME" \
+        -adminpassword "$ADMIN_PASSWORD" \
+        -ip "$BIND_IP" -port "$QUERY_PORT" \
+        -servername "$SERVER_NAME" \
+        -steamvac "$STEAM_VAC" "$USE_STEAM"
+}
```

Comment:
This is a good idea for generating the server files before starting, but I think (I may be wrong) I solved this in a recent commit so that it shouldn't be necessary to start the server twice.  It may be required for mods though, and I just don't know yet.  

I will be pulling down your changes locally to verify how everything works anyway.

Reply:
Server files were not created until server start in my testing unfortunalty

Reply:
This was correct, I can see now that the initial run of the server doesn't have the necessary files in the `$CONFIG_DIR` directory because they hadn't yet been created.

I actually have this fixed now, but I am unsure how to merge our work.  Would you mind if I made a new PR from your changes with my modifications?

Reply:
Sounds good to me